Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -49,9 +78,11 @@ public void test_getReview_success() {
         reviewRequest.setConceptId("WID-10126926-N-05-Einstein");
         reviewRequest.setRequest("Testing Method2");
         
-        Mockito.when(dbClient.getReviewRequestForConcept("WID-10126926-N-05-Einstein")).thenReturn(reviewRequest);
+        List<ReviewRequest> response = new ArrayList<>();
+        response.add(reviewRequest);
+        Mockito.when(dbClient.getAllReviews("WID-10126926-N-05-Einstein")).thenReturn(response);
 
-        ReviewRequest fetchedReview = requestManager.getReview("WID-10126926-N-05-Einstein");
+        ReviewRequest fetchedReview = requestManager.getLatestReview("WID-10126926-N-05-Einstein");
```

Comment:
test method needs to be renamed according to tested method

Reply:
also, the important thing here is that getLatestReview gives you the most recent review. You are not testing that.